cmd/state-installer/cmd.go

```diff
@@ -200,11 +201,9 @@ func main() {
 			multilog.Critical("Installer error: " + errs.JoinMessage(err))
 		}
 
-		exitCode = errs.UnwrapExitCode(err)
+		exitCode, err = errors.Unwrap(err)
 		an.EventWithLabel(AnalyticsFunnelCat, "fail", err.Error())
-		if !errs.IsSilent(err) {
-			out.Error(err.Error())
-		}
```

Comment:
This part still seems relevant. This was handling the case where we run the installer with a subshell, and then something the user did in the subshell errored. This is not an error on our end so we should not print it.

Reply:
Oh, I erroneously thought that was handled here: https://github.com/ActiveState/cli/blob/8ed0bef228d492ce2203e325226f80fd84ed3268/pkg/cmdlets/errors/errors.go#L94